src/components/DigitalCredentials/IssueCredentials.vue

```diff
@@ -0,0 +1,130 @@
+              register a digital wallet.
+            </span>
+          </p>
+        </v-col>
+      </v-row>
+    </article>
+
+    <article v-if="hasRegisteredWallet">
+      <v-row no-gutters>
+        <v-col cols="12" lg="12">
+          <v-card flat rounded class="px-2 mt-3">
+            <v-card-text class="px-6 py-8">
+
+              <!-- Credential Offer -->
+              <v-row no-gutters>
+                <v-col cols="12" lg="8" class="ma-auto">
+                  <h4>Business Digital Credential</h4>
+                </v-col>
+                <v-col cols="12" lg="4" class="pr-2">
+                  <v-btn
+                    v-if="issuedCredentials.length === 0"
+                    id="next-btn"
+                    large color="primary"
+                    class="font-weight-bold px-6 float-right"
+                    :loading="isSendingOffer"
+                    @click="sendCredentialOffer(DigitalCredentialTypes.BUSINESS)"
+                  >Send Credential Offer</v-btn>
+
+                  <span v-else-if="isPendingAcceptedOffer" class="font-weight-bold alert-text">
+                    <v-icon class="pr-2 alert-text">mdi-wallet</v-icon>
+                    Check Digital Wallet for Offer
+                  </span>
+
+                  <span v-else class="success-text font-weight-bold">
+                    <v-icon color="success" class="pr-2">mdi-check</v-icon>
+                    Credential Successfully Stored
+                  </span>
+                </v-col>
+              </v-row>
+
+            </v-card-text>
+          </v-card>
+        </v-col>
+      </v-row>
+    </article>
+
+    <!-- Navigate back section when no selections made -->
+    <article v-else id="no-credentials-msg" @click="back()">
+      <v-card flat rounded class="pa-3">
+        <v-card-text>
+          <v-row no-gutters>
+            <v-col cols="12">
+              <span class="font-16">
+                <v-icon class="pr-1" color="primary">mdi-arrow-left</v-icon>
+                Go back to the previous step to register a digital wallet with this business.
+              </span>
+            </v-col>
+          </v-row>
+        </v-card-text>
+      </v-card>
+    </article>
+  </section>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Prop, Vue } from 'vue-property-decorator'
+import { DigitalCredentialTypes } from '@/enums'
+import { DigitalCredentialsIF } from '@/interfaces'
+
+@Component({})
+export default class IssueCredentials extends Vue {
+  @Prop({ default: false })
+  readonly hasRegisteredWallet: boolean
+
+  @Prop({ default: false })
+  readonly issuedCredentials: Array<DigitalCredentialsIF>
+
+  readonly DigitalCredentialTypes = DigitalCredentialTypes
+  private isSendingOffer = false
+
+  get isPendingAcceptedOffer (): boolean {
+    return !this.issuedCredentials[0]?.isIssued
+  }
+
+  @Emit() back (): void {}
```

Comment:
Please change this to `@Emit() protected back (): void {}`.
(Consistent with the other changes in my PR.)

Reply:
Sounds good, updated!